vizro-ai/docs/pages/user-guides/api-setup.md

```diff
@@ -1,17 +1,20 @@
 # How to setup the API
+
 This guide shows how to set up the API key and other environment variables (e.g.,
 base URL) so that you can seamlessly communicate with the Large Language Model (LLM).
 
 Users are recommended to review the third party API key section of the [disclaimer](../explanation/disclaimer.md) documentation.
 
 ## OpenAI
+
 To communicate with the OpenAI API, you need to provide the API key. Additionally, you
 might need to provide the base URL if you are using a custom OpenAI resource endpoint.
 
 ### Setup the API key
+
 There are two common ways to set up the API key in a development environment.
 
-__Method 1 (recommended): Set up the API key for a single project__
+**Method 1 (recommended): Set up the API key for a single project**
```

Comment:
This is so strange, I didn't change this!

Reply:
Ahhhh, I know why 😅 Our automatic pre-commit auto-fix did that there: https://github.com/mckinsey/vizro/pull/320/commits/80db16e82c7870ec9d366ae0a3b85d93747782f6


The reason for that was that we ignore these folders here, but because you've changed the names, they didn't get ignored anymore. So you would also have to update this file:

https://github.com/mckinsey/vizro/blob/main/.prettierignore

Reply:
Oh, that makes sense, thanks for diagnosing that for me! I shall fix that....and then go through and fix any broken links I've made in the content. Should be ready for a second review today!

Reply:
Let me know if you need help with reverting the auto-fix commit and/or the changelog files :)